liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -63,12 +65,15 @@ protected void doExecute() throws Exception {
             installDependencies();
         }
         if (installProject) {
-            installApp(project.getArtifact());
+            installProject();
         }
     }
     
     private void installDependencies() throws Exception {
-        for (Artifact dep : (Set<Artifact>) project.getDependencyArtifacts()) {
+        @SuppressWarnings("unchecked")
+        Set<Artifact> artifacts = (Set<Artifact>) project.getDependencyArtifacts();
+        for (Artifact dep : artifacts) {
+            // TODO: skip if not an application type supported by Liberty
```

Comment:
I don't see anything being done in the loop.  Sort of confused by the TODO message.